src/App.jsx

```diff
@@ -9,22 +9,22 @@ import FormForTestWithImage from './components/FormForTestWithImage/FormForTestW
 import ProductsPage from './pages/ProductsPage/ProductsPage.jsx';
 
 const App = () => {
-  return (
-    <>
-      <Routes>
-        <Route path="" element={<MainLayout />}>
-          <Route path="/" element={<HomePage />} />
-          <Route path="/category/:name" element={<ProductsPage />} />
-        </Route>
+    return (
```

Comment:
There are very long spaces. Our prettier has another spacing preferences "tabWidth: 2" and other.